### ISSUE TITLE
Does this support/include packages?

### ISSUE THREAD

**Issue by koolskateguy89** [mfuentesg/SyncSettings#187]
As in will this enable me to easily reinstall my packages on another device?

**Comment by mfuentesg** [comment]
@koolskateguy89  what do you want to say?

**Comment by koolskateguy89** [comment]
I understand it allows me to keep my settings the same across devices, but does this also include my packages list?

**Comment by mfuentesg** [comment]
Yeah, it includes your package list defined in your package control settings file

**Comment by koolskateguy89** [comment]
That's brilliant thank you
